packages/widgetbook/lib/src/addons/grid_addon/grid_addon.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+import '../../fields/fields.dart';
+import '../common/common.dart';
+import 'grid_painter.dart';
+import 'grid_setting.dart';
+
+class GridAddon extends WidgetbookAddon<GridSetting> {
+  GridAddon([int? size])
+      : super(
+          name: 'Grid',
+          initialSetting: GridSetting(size: size ?? 5),
+        );
+
+  @override
+  List<Field> get fields => [];
+
+  @override
+  GridSetting valueFromQueryGroup(Map<String, String> group) {
+    return GridSetting(
+      size: int.tryParse(group['size'] ?? '') ?? initialSetting.size,
+    );
+  }
```

Comment:
We can use a constant `dimension` field that will be used in all overrides as follows:

```suggestion
class GridAddon extends WidgetbookAddon<int> {
  GridAddon([this.dimension = 48])
      : super(
          name: 'Grid',
          initialSetting: dimension,
        );

  final int dimension;

  @override
  List<Field> get fields => [];

  @override
  int valueFromQueryGroup(Map<String, String> group) => dimension;
```